libs.autobuild

```diff
@@ -601,6 +601,12 @@ in_flavor 'master', 'stable' do
     remove_from_default 'external/kdtree'
     Autoproj.add_osdeps_overrides 'libkdtree', :package => 'external/kdtree'
 
+    cmake_package 'external/udt' do |pkg|
+
+    end
```

Comment:
You don't need the block.

Also, we're trying to phase out external (for years now ...). Can you place it elsewhere ?

Reply:
How about multiagent/udt? would put it in the same place as the package depending on udt.

Reply:
What's UDT ?

Reply:
Their readme describes it as "UDP-based Data Transfer (UDT) Library - version 4", which may also point towards tools/

Reply:
I would definitely put that in `tools/`

Reply:
moved to tools/udt, removed the extra do-block, moved the declaration up in libs.autobuild, as indicated by numerous comments in there.

Reply:
@doudou please merge if acceptable